include/boost/unordered/detail/foa/concurrent_table.hpp

```diff
@@ -88,7 +88,7 @@ class cache_aligned_array
   static constexpr std::size_t element_offset=
     (sizeof(T)+cacheline_size-1)/cacheline_size*cacheline_size;
 
-  BOOST_STATIC_ASSERT(alignof(T)<=cacheline_size);
+  BOOST_UNORDERED_STATIC_ASSERT(alignof(T)<=cacheline_size);
```

Comment:
Sure; there's no need in C++11 and above to include boost/static_assert.hpp. We only need a one-line macro and that only because we don't supply error messages in the assertions.

Reply:
Well, StaticAssert is pretty small so it's not the end of the world but in general, I'd prefer to not introduce an extra dependency for a native language feature. A goal I had in mind when doing this refactor was minimizing the output of `boostdep` so in turn this will reduce our dependency graph with package managers too.

Assuming Conan ever figures out modular Boost.